corehq/apps/reports/static/reports/js/filters/case_list_explorer_knockout_bindings.js

```diff
@@ -1,42 +1,44 @@
-hqDefine("reports/js/filters/case_list_explorer_knockout_bindings", ['jquery', 'knockout', 'hqwebapp/js/atwho'], function($, ko, atwho) {
+hqDefine("reports/js/filters/case_list_explorer_knockout_bindings", ['jquery', 'knockout', 'hqwebapp/js/atwho', 'ace-builds/src-min-noconflict/ace'], function($, ko, atwho) {
```

Comment:
Curious why you picked this syntax highlighter in particular and how it compares to the two we already use (CodeMirror and SyntaxHighlighter). Eventually, I'd like to standardize on one.